frontend/src/types/mapbox.ts

```
@@ -1,59 +1,117 @@
-import mapboxgl from 'mapbox-gl';
+import {
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)